.circleci/config.yml

```diff
@@ -613,9 +613,6 @@ workflows:
   build:
```

Comment:
FYI: You can use `.gitattribute` file to tell git that this is a generated file. That way, it will be hidden in PR by default.

https://github.com/pytorch/audio/blob/main/.gitattributes

Reply:
That's a good callout . Will send you a PR shortly!